api/lib/infrastructure/utils/csv/certification-results.js

```diff
@@ -16,8 +16,20 @@ async function getSessionCertificationResultsCsv({
   session,
   certificationResults,
 }) {
-  const data = _buildCertificationResultsFileDataWithCertificationCenterName({ session, certificationResults });
-  const fileHeaders = _buildCertificationResultsFileHeadersWithCertificationCenterName();
+
+  const doesAtLeastOneCertificationHasPassedClea = certificationResults.some((result) =>
+    result.cleaCertificationStatus === 'aquired' || result.cleaCertificationStatus === 'rejected');
```

Comment:
typo sur 
```suggestion
    result.cleaCertificationStatus === 'acquired' || result.cleaCertificationStatus === 'rejected');
```
d'où la nécessité de toujours passer par les structures de données dédiées. Vous l'avez fait correctement dans le test mais pas dans le code de prod :
```js
const { statuses: cleaStatuses } = require('(../)*lib/infrastructure/repositories/clea-certification-status-repository');

/*...*/
result.cleaCertificationStatus === cleaStatuses.REJECTED || result.cleaCertificationStatus === cleaStatuses.ACQUIRED);

```

Reply:
Je propose aussi ceci comme réécriture :
```js
const doesAtLeastOneCertificationHasPassedClea = certificationResults.some((result) =>
    [cleaStatuses.REJECTED, cleaStatuses.ACQUIRED].includes(result.cleaCertificationStatus));
```

Reply:
Ou bien y'a ça mais c'est moins lisible je trouve :
```js
const doesAtLeastOneCertificationHasPassedClea = !_.every(certificationResults, { 'cleaCertificationStatus': cleaStatuses.NOT_PASSED });
```